src/components/side_nav/_side_nav_item.scss

```
@@ -148,12 +148,9 @@
   & > .euiSideNavItem__items {
     margin-left: $euiSize;
   }
-}
 
-.euiSideNavItem--hasChildItems {
-  & > .euiSideNavItemButton-isOpen {
-    .euiSideNavItemButton__label {
-      color: $euiColorFullShade;
-    }
+  .euiSideNavItemButton__label {
+    color: $euiColorDarkShade;
```

Then lets use the subdued text color here

```suggestion
    color: $euiTextSubduedColor;
```